cypress/e2e/0-api/transmitter_api.cy.js

```diff
@@ -0,0 +1,236 @@
+          expect(body.serialNumber).to.be.a('string');
+        });
+      });
+    });
+  });
+
+  context('GET /transmitters/local/latest_reading', () => {
+    it('retrieves latest reading on transmitter', () => {
+      cy.reqLatestTransmitterReading().then(response => {
+        const body = response.body;
+        expect(body.timestamp).to.be.a.dateString();
+        expect(body.inputVoltage).to.be.a('number');
+        expect(body.backplaneVoltage).to.be.a('number');
+        expect(body.totalCurrent).to.be.a('number');
+        expect(body.temperature).to.be.a('number');
+        expect(body.mgtStatus).to.be.a('number');
+        expect(body.txFaultSummary).to.be.a('number');
+        expect(body.flags).to.be.a('number');
+      });
+    });
+  });
+
+  context('GET /transmitters/local/readings', () => {
+    it('retrieves latest n readings on transmitter', () => {
+      const NUMBER_OF_READINGS = 5;
+      cy.reqTransmitterReadingsByLatest(NUMBER_OF_READINGS).then(response => {
+        const body = response.body;
+        body.forEach(reading => {
+          expect(reading.timestamp).to.be.a.dateString();
+          expect(reading.inputVoltage).to.be.a('number');
+          expect(reading.backplaneVoltage).to.be.a('number');
+          expect(reading.totalCurrent).to.be.a('number');
+          expect(reading.temperature).to.be.a('number');
+          expect(reading.mgtStatus).to.be.a('number');
+          expect(reading.txFaultSummary).to.be.a('number');
+          expect(reading.flags).to.be.a('number');
+        });
+      });
+    });
+    it('retrieves readings between range of dates on transmitter', () => {
+      // Had to nest API calls because global variables could not be reassigned from within a Cypress call
+      cy.reqLatestTransmitterReading().then(response => {
+        const body = response.body;
+        const DATE = body.timestamp;
+        cy.log(DATE);
+        const START = DATE;
+        // Increments date by 1 hour
+        const END = DATE.substring(0, 12) + (parseInt(DATE.substring(12, 13)) + 1) + DATE.substring(13);
+        cy.reqTransmitterReadingsByRange(START, END).then(response => {
+          const body = response.body;
+          body.forEach(reading => {
+            expect(reading.timestamp).to.be.a.dateString();
+            expect(reading.inputVoltage).to.be.a('number');
+            expect(reading.backplaneVoltage).to.be.a('number');
+            expect(reading.totalCurrent).to.be.a('number');
+            expect(reading.temperature).to.be.a('number');
+            expect(reading.mgtStatus).to.be.a('number');
+            expect(reading.txFaultSummary).to.be.a('number');
+            expect(reading.flags).to.be.a('number');
+          });
+        });
+      });
+    });
+  });
+
+  // TODO: Test out watch dog parameter
+  context('GET /transmitters/local/live_data', () => {
+    it('retrieves and checks transmitter data', () => {
```

Comment:
'it' strings should describe what the feature is/does, not what the test does.  i.e.

```suggestion
    it('returns transmitter data', () => {
```